modules/logging-loki-storage-azure.adoc

```diff
@@ -1,17 +1,20 @@
 // Module is included in the following assemblies:
-// logging/cluster-logging-loki.adoc
+// logging/log_storage/installing-log-storage.adoc
 //
 :_mod-docs-content-type: PROCEDURE
 [id="logging-loki-storage-azure_{context}"]
 = Azure storage
```

Comment:
Procedure titles should begin with a gerund, such as "Configuring Azure storage".

Reply:
Out of scope